### PR TITLE
first draft

### PR DESCRIPTION
**Pull request by mekolat**
this is my first draft using the most recent version of the tiled [npc proposal](https://www.themanaworld.org/index.php/User:Meko/Tiled/NPC_Proposal) for Hercules

this is only a [DIS] and is not intended to be merged

### CODE REVIEW

File: maps/029-2.tmx [hunk 1]
```diff
@@ -944,5 +944,189 @@
...
+  <object id="24" name="#CandorBank#Door" type="npc" x="1120" y="3904" width="32" height="32">
+   <properties>
+    <property name="callfunc" value="CandorBank_Door"/>
+    <property name="sprite" value="45"/>
+    <property name="trigger" value="0,0"/>
+   </properties>
+  </object>
+  <object id="25" name="Barrel#0" type="npc" x="736" y="2752" width="32" height="32">
+   <properties>
+    <property name="arguments" value="0"/>
```
Comment (mekolat) [inline maps/029-2.tmx]: @wushin the parser should be able to tell the difference between lack of arguments and 0

Comment (wushin) [inline maps/029-2.tmx]: yes. Null and 0 are 2 different things

File: maps/029-2.tmx [hunk 2]
```diff
@@ -944,5 +944,189 @@
...
+  </object>
+  <object id="41" name="Tanisha" type="npc" x="3520" y="2816" width="32" height="32">
+   <properties>
+    <property name="callfunc" value="Tanisha"/>
+    <property name="sprite" value="114"/>
+   </properties>
+  </object>
+  <object id="42" name="Rosen" type="shop" x="3168" y="800" width="32" height="32">
+   <properties>
+    <property name="items" value="SlingShot,SlingBullet,ShortBow:*4,Arrow:*2,IronArrow"/>
```
Comment (wushin) [inline maps/029-2.tmx]: Make items value include :*1 as str is a kludge to parse.

File: maps/029-2.tmx [hunk 3]
```diff
@@ -944,5 +944,189 @@
...
    </properties>
   </object>
+  <object id="10" name="Sorfina" type="npc" x="864" y="864" width="32" height="32">
+   <properties>
+    <property name="callfunc" value="Sorfina"/>
+    <property name="sprite" value="154"/>
+    <property name="trigger" value="0,1"/>
+   </properties>
+  </object>
+  <object id="11" name="#" type="npc" x="1056" y="864" width="32" height="32">
```
Comment (wushin) [inline maps/029-2.tmx]: The NPCs should be named so we can determine their function visible in tiled.

**Pull request closed without merging**